WooCommerce/Classes/ViewRelated/Dashboard/Stats v4/StoreStatsV4PeriodViewController.swift

```diff
@@ -242,11 +243,14 @@ private extension StoreStatsV4PeriodViewController {
     }
 
     func configureView() {
-        view.backgroundColor = StyleManager.wooWhite
-        borderView.backgroundColor = StyleManager.wooGreyBorder
+        view.backgroundColor = .listForeground
+        containerStackView.backgroundColor = .listForeground
+        timeRangeBarView.backgroundColor = .listForeground
+        visitorsStackView.backgroundColor = .listForeground
+        borderView.backgroundColor = .listSmallIcon
 
         // Time range bar bottom border view
-        timeRangeBarBottomBorderView.backgroundColor = StyleManager.wooGreyBorder
+        timeRangeBarBottomBorderView.backgroundColor = .listForeground
```

Comment:
I've started using `.divider` for border views, not sure if we should use `.divider` or `.listForeground` for the borders?